src/index.html

```diff
@@ -7,8 +7,60 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;500;900&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <!-- <h1>Moyo header</h1> -->
+    <header class="container">
+      <a href="#home" class="logo"><img src="images/logo.png" alt="/"></a>
+      <nav class="nav">
+        <ul class= "navList">
+          <li class="link">
+            <a href="#" class="nav--link is-active">
+              Apple
+            </a>
+          </li>
+          <li class ="link">
```

Comment:
```suggestion
          <li class="link">
```
no need to leave space here